packages/vscode-extension/src/builders/BuildManager.ts

```diff
@@ -71,19 +71,25 @@ export class BuildManager {
         this.buildOutputChannel = window.createOutputChannel("Radon IDE (iOS build)", {
           log: true,
         });
+        const appRootFolder = getAppRootFolder();
+        const installPodsIfNeeded = async () => {
+          const podsInstalled = await this.dependencyManager.checkPodsInstalled();
+          if (!podsInstalled) {
+            Logger.info("Pods installation is missing or outdated. Installing Pods.");
+            // installing pods may impact the fingerprint as new pods may be created under the project directory
+            // for this reason we need to recalculate the fingerprint after installing pods
+            await this.dependencyManager.installPods(appRootFolder, cancelToken);
```

Comment:
I wonder if it would be easier if we moved this function to be inside dependency manager (same goes for installNodeModules(ifNeeded)) and returned new fingerprint.

Reply:
I'd keep it as is for now. We only call it from one place right now